pluribus/games/short_deck/state.py

```diff
@@ -272,9 +272,7 @@ def initial_regret(self) -> Dict[str, float]:
     @property
     def initial_strategy(self) -> Dict[str, float]:
         """Returns the default strategy for this state."""
-        actions = self.legal_actions
-        equal_probability = 1 / len(actions)
-        return {action: equal_probability for action in actions}
+        return {action: 0 for action in self.legal_actions}
```

Comment:
Still making my way through the changes, but right now I'm not sure why you changed this to 0